tasks/update_record.json

```diff
@@ -1,21 +1,21 @@
 {
     "puppet_task_version": 1,
     "supports_noop": false,
-    "description": "Resolve an incident",
+    "description": "Update a ServiceNow record",
     "files": ["ruby_task_helper/files/task_helper.rb"],
     "remote": true,
     "parameters": {
-        "sys_id": {
-            "description": "ServiceNow sys_id",
+        "table": {
+            "description": "ServiceNow table",
             "type": "String[1]"
         },
-        "close_notes": {
-            "description": "ServiceNow close notes",
+        "sys_id": {
+            "description": "ServiceNow sys_id",
             "type": "String[1]"
         },
-        "additional_data": {
-            "description": "ServiceNow additional data",
-            "type": "String[1]"
+        "fields": {
+            "description": "ServiceNow fields. This is a hash of <field_name> => <value>",
```

Comment:
I'd maybe change "ServiceNow fields ..." => "The updated fields. ...." Latter provides a bit more context.